libcontainer/configs/config.go

```diff
@@ -33,9 +33,10 @@ type Seccomp struct {
 type Action int
 
 const (
-	Kill Action = iota - 4
```

Comment:
This way, uninitialized Action variables will not be a valid constant, and will cause an error